packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -1129,6 +1137,14 @@ export class AzureCommunicationCallAdapter<AgentType extends CallAgent | BetaTea
     this.emitter.emit('transferRequested', newArgs);
   }
 
+  /* @conditional-compile-remove(capabilities) */
+  private capabilitiesChanged(data: CapabilitiesChangeInfo): void {
+    if (data.newValue.turnVideoOn?.isPresent === false) {
+      this.stopCamera();
```

Comment:
Should we also be looking at the microphone state here to? Mute the participant if the capability is removed?

Reply:
The calling SDK mutes the mic when the unmute capability changed to not present but yeah we also make sure on our side.